src/test/ui/pattern/const-pat-ice.stderr

```diff
@@ -14,4 +14,4 @@ note: compiler flags: FLAGS
 query stack during panic:
 #0 [check_match] match-checking `main`
 #1 [analysis] running analysis passes on this crate
-we're just showing a limited slice of the query stack
+we're just showing a limited slice of the query stack
```

Comment:
I think this will cause bors to fail again, is there a reason you removed it?

Reply:
you mean about new line?

Reply:
Yeah, you removed the newline.